src/coalescenceml/integrations/tensorflow/step/tensorflow_trainstep.py

```diff
@@ -0,0 +1,77 @@
+# https://www.youtube.com/watch?v=cJ3oqHqRBF0
+class TFClassifierTrainStep(BaseStep):
+    # Hyperparameters used in video: epochs 256, batch_size 10
+    # Note to users: layers parameter shouldn't have 1 as last - it is automatically done
+    def entrypoint(
+        self,
+        x_train: np.ndarray,
+        y_train: np.ndarray,
+        layers: typing.List,
+        hyperparams: typing.Dict = {},
+        x_validation: np.ndarray = np.array([]),
+        y_validation: np.ndarray = np.array([]),
+    ) -> tf.keras.Model:
+        # Possibly import optimizer and change its hyperparameters
+
+        model = Sequential()
+
+        # Construct layers
+        for x in range(len(layers)):
+            if x == 0:
+                model.add(
+                    Dense(
+                        layers[x],
+                        input_dim=len(x_train[0, :]),
+                        activation="relu",
+                    )
+                )
+            else:
+                model.add(Dense(layers[x], activation="relu"))
+        model.add(Dense(1, activation="sigmoid"))
+
+        model.compile(
+            loss="binary_crossentropy", optimizer="adam", metrics=["accuracy"]
```

Comment:
Is the model only trained on binary crossentropy? What about for other losses and optimizers?